CHANGES.md

```diff
@@ -4,6 +4,9 @@
 - Fix a bug affecting bound classmethod saving on Python 2.
   ([issue #288](https://github.com/cloudpipe/cloudpickle/issues/288))
 
+- Add support for pickling exotic builtin property types.
```

Comment:
Let's be more objective and descriptive: "add support for pickling "getset" descriptors.